packages/sdk/src/signature.ts

```diff
@@ -0,0 +1,29 @@
+import { Secp256k1 } from "@iov/crypto";
+import { Encoding } from "@iov/encoding";
+
+import { encodeSecp256k1Pubkey } from "./pubkey";
+import { pubkeyType, StdSignature } from "./types";
+
+export function encodeSecp256k1Signature(pubkey: Uint8Array, signature: Uint8Array): StdSignature {
+  return {
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    pub_key: encodeSecp256k1Pubkey(pubkey),
+    // Recovery seems to be unused
+    signature: Encoding.toBase64(Secp256k1.trimRecoveryByte(signature)),
+  };
+}
+
+export function decodeSignature(
```

Comment:
This one is new and supposed to be more or less the inverse of `encodeSecp256k1Signature`